gobblin-data-management/src/main/java/gobblin/data/management/retention/dataset/ConfigurableCleanableDataset.java

```diff
@@ -96,6 +106,14 @@
   private final List<VersionFinderAndPolicy<T>> versionFindersAndPolicies;
 
   /**
+   * A set of all available {@link RetentionActionFactory}s
+   */
+  private static final Set<Class<? extends RetentionActionFactory>> RETENTION_ACTION_TYPES;
```

Comment:
`Alias` is not required for `RetentionActionFactory` because it is not injected by the user.